xdsl/dialects/llvm.py

```diff
@@ -91,6 +92,79 @@ def opaque():
     def typed(type: Attribute):
         return LLVMPointerType([type, NoneAttr()])
 
+    def is_typed(self):
+        return not isinstance(self.type, NoneAttr)
+
+
+@irdl_op_definition
+class GEPOp(Operation):
+    name = "llvm.getelementptr"
+
+    ptr: Annotated[Operand, LLVMPointerType]
+    ssa_indices: Annotated[VarOperand, IntegerType]
+
+    elem_type: OptOpAttr[Attribute]
+
+    rawConstantIndices: OpAttr[DenseArrayBase]
+
+    inbounds: OptOpAttr[UnitAttr]
+
+    result: OpResult
+
+    @staticmethod
+    def get(ptr: SSAValue | Operation,
+            indices: list[int] | None = None,
+            ssa_indices: list[SSAValue | Operation] | None = None,
+            result_type: Attribute | None = None,
+            inbounds: bool = False,
+            ptr_type: Attribute | None = None):
+        # construct default mutable argument here:
+        if indices is None:
+            indices = []
+        # TODO: convert indices to DenseI32ArrayAttr
```

Comment:
This is left over, right?

Reply:
Yes whoops

Reply:
It's being used as a way to create the indices_attr

Reply:
Should I do it a different way? Less variables?

Reply:
i was talking about the comment :)

Reply:
Oooo, yes, yes redundant